sources/packages/web/src/views/institution/locations/programs/LocationProgramView.vue

```diff
@@ -61,7 +146,39 @@ export default {
       });
     };
 
-    return { goToEditProgram, goToAddNewOffering };
+    const goToEditOffering = (offeringId: number) => {
+      router.push({
+        name: InstitutionRoutesConst.EDIT_LOCATION_OFFERINGS,
+        params: {
+          offeringId: offeringId,
+          programId: props.programId,
+          locationId: props.locationId,
+        },
+      });
+    };
+
+    const offerings = ref([] as EducationProgramOfferingDto[]);
+    const educationProgram = ref({} as EducationProgramDto);
+    const getEducationProgramAndOffering = async () => {
+      offerings.value = await EducationProgramOfferingService.shared.getAllEducationProgramOffering(
```

Comment:
Please change the 2 API calls to be in parallel using Promise.all.

Reply:
done